tests/sentry/sentry_metrics/querying/data_v2/test_api.py

```diff
@@ -93,6 +102,28 @@ def to_reference_unit(
         _, _, unit = unit_family_and_unit
         return unit.convert(value)
 
+    def run_query(
```

Comment:
Abstracted all function calls behind this, so that we could change the transformer at will in case we decide to directly read the data from the raw results.